docs/templates/python/material/attribute.html

```diff
@@ -1,9 +1,10 @@
 {# Modified from https://github.com/mkdocstrings/python/blob/master/src/mkdocstrings_handlers/python/templates/material/_base/attribute.html #}
+{# Updated 2024/05/20. See "START NEW CODE" for block that is new. #}
```

Comment:
So the real necessary change here was to `git mv` the file. But while I was at it I also took the latest version from the URL above. We'll see if it renders okay. If not we can revert